### PR TITLE
Working etcd example application (flows example)

### PR DESCRIPTION
**Pull request by istalker2**
This change is [Reviewable](https://reviewable.io/reviews/mirantis/k8s-appcontroller/246)

### CODE REVIEW

**Comment by pigmej** [comment]
@istalker2 what would be the easiest way to validate it?

File: examples/etcd/resdefs/bootstrap-flow.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: appcontroller.k8s/v1alpha1
+kind: Flow
```
Comment (nebril) [inline examples/etcd/resdefs/bootstrap-flow.yaml]: Are flows stored in etcd? I thought that they were only "virtual resources" without new adding new TPR.

File: examples/etcd/create.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# /bin/sh
```
Comment (istalker2) [inline examples/etcd/create.sh]: what is wrong with hashbang? Aren't all shell scripts suppose to start with it?

File: examples/etcd/create.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+#!/bin/bash
+
+source ../common.sh
+
+set -x
+
+for file in resdefs/*.yaml
+do 
+  cat $file | $KUBECTL_NAME exec -i k8s-appcontroller kubeac wrap | $KUBECTL_NAME create -f-
```
Comment (istalker2) [inline examples/etcd/create.sh]: okay. But note, that all other examples have it hardcoded

File: examples/etcd/resdefs/bootstrap-flow.yaml (hunk 1 above)
Comment (istalker2) [inline examples/etcd/resdefs/bootstrap-flow.yaml]: flows need to be wrapped first

File: examples/etcd/build-image.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+docker build . -t etcd-custom
```
Comment (istalker2) [inline examples/etcd/build-image.sh]: ok

**Comment by istalker2** [comment]
fixed

**Comment by pigmej** [comment]
As a second step we will need to think how to get rid of modifying etcd image .

**Pull request merged**